Paralayout/UIView+Alignment.swift

```diff
@@ -19,16 +19,27 @@ import UIKit
 
 extension UIView {
 
-    // MARK: - View Alignment - Core
+    // MARK: - Public Types
+
+    public enum TargetAlignmentBehavior {
+
+        /// Align to the position in the target view's bounds, ignoring any origin offset. This is most commonly used
+        /// when aligning sibling views in a hierarchy.
+        case untransformedFrame
```

Comment:
I think there's likely a more accurate name for this, but I can't think of one at the moment.

Reply:
this is a tough one to name😅
What do you think about:
.boundsWithoutOffset

Reply:
Maybe... I think it might actually be better to name it by the intended usage, e.g. `interior`/`exterior`. I'm gonna go ahead and merge this and file an issue to think about a better name before the 1.0 release.